converters/toZigbee.js

```diff
@@ -239,6 +239,93 @@ const converters = {
             await entity.read('msOccupancySensing', ['pirOToUDelay']);
         },
     },
+    level_config: {
+        key: ['level_config'],
+        convertSet: async (entity, key, value, meta) => {
+            const state = {};
+
+            // parse payload to grab the keys
+            try {
+                value = JSON.parse(value);
+            } catch (e) {
+                e;
```

Comment:
I think we should throw an error in this case.

Reply:
So just remove the try-catch? Or leave as is but log an error via meta.logger ?

Reply:
throw a friendly error here, e.g. `throw new Error("Payload is not valid JSON")`